CHANGELOG.md

```diff
@@ -0,0 +1,290 @@
+
+## [3.1.4][] - 2017-03-20
+
+Fixes a bug where text in a static portlet was hard to read (#380)
+
+## [3.1.3][] - 2017-03-09
+
+Adds back in UW-Lacrosse theming
+
+## 3.1.2 - \[Yanked]
+
+## [3.1.1][] - 2017-03-02
+
+Increases Notification documentation #375
+Adds more documentation about best practices for notifications #376
+Adjusts link colors for themes #377
+
+## [3.1.0][] - 2017-02-21
+
+### Uses GitHub Pages
+
+[Adopts GitHub pages](https://github.com/UW-Madison-DoIT/uw-frame/pull/361)
+[Fixes documentation bugs](https://github.com/UW-Madison-DoIT/uw-frame/pull/363)
+[Normalizes headings in releasing.md](https://github.com/UW-Madison-DoIT/uw-frame/pull/364)
+[Continues documentation cleanup](https://github.com/UW-Madison-DoIT/uw-frame/pull/366)
+[Normalizes Jekyll theme to match angularjs-portal Jekyll Theme](https://github.com/UW-Madison-DoIT/uw-frame/pull/368)
+[Fixes documentation link bugs](https://github.com/UW-Madison-DoIT/uw-frame/pull/369)
+
+### Continue support of Windows build environments
+
+[Adds Windows build support](https://github.com/UW-Madison-DoIT/uw-frame/pull/360)
+[Adds font-awesome into the Windows build](https://github.com/UW-Madison-DoIT/uw-frame/pull/370)
+
+### Removal of UW specific wording
+
+[Removes hardcoding of UW instances](https://github.com/UW-Madison-DoIT/uw-frame/pull/372)
+[Removes the assumption that all helpdesks are in DoIT](https://github.com/UW-Madison-DoIT/uw-frame/pull/374)
+
+### Misc
+
+[Modernize License declarations](https://github.com/UW-Madison-DoIT/uw-frame/pull/365)
+[Fixes launch button shadowing bug](https://github.com/UW-Madison-DoIT/uw-frame/pull/362)
+[Adds codeclimate badges](https://github.com/UW-Madison-DoIT/uw-frame/pull/359)
+[Adds dependenci badges](https://github.com/UW-Madison-DoIT/uw-frame/pull/358)
+
+## [3.0.3][] - 2016-12-20
+
+Patch release. Simply bump your app's dependency declaration from `uw-frame` `3.0.2` to `3.0.3` to adopt this release.
+- Fixes Google Analytics usage ( #353 )
+- Documents source code whitespace conventions ( #356 )
+- Improves [documentation about releasing `uw-frame` itself](http://uw-madison-doit.github.io/uw-frame/v3.0.3/#/md/releasing) ( #355 )
+
+See also
+- [the 3.0.3 milestone](https://github.com/UW-Madison-DoIT/uw-frame/milestone/9?closed=1).
+- [release announcement](https://groups.google.com/forum/#!topic/myuw-developers/evV8Ie3AhfQ).
+
+## [3.0.2][] - 2016-12-14
+
+Patch release. Trivial upgrade from `v3.0.1` (or even `v3.0.0`).
+- Username menu items are no longer redundantly duplicatively labeled ( #351 )
+- Circle buttons now maintain their circular shape even when their label text wraps ( #352 )
+
+Supporting links:
+- [Documentation for this version specifically](http://uw-madison-doit.github.io/uw-frame/v3.0.2/).
+- [Release announcement on myuw-developers group](https://groups.google.com/d/topic/myuw-developers/tKOWjo8r96c/discussion)
+
+## 3.0.1 - \[YANKED]
```

Comment:
:question: any details on why this release was skipped or yanked that could be added?

Reply:
My guess is that something went wrong with the release to Maven central, so the git tag was yanked, and the release process tried again.

I'm not 100% sure, but I'm 95% confident that's what happened.

Reply:
Thanks @vertein, I updated the changelog with the correct date for yanked releases as well as a note on maven central issues.